From 4fd40705074f5bc90999f4d3dc4f28ad5bd5f7c5 Mon Sep 17 00:00:00 2001
From: PengchengYang-xdu <24241214835@stu.xidian.edu.cn>
Date: Wed, 8 Oct 2025 17:03:09 +0800
Subject: [PATCH 71/72] ci commit

---
 .gitignore | 1 +
 Makefile   | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index 3c0160d0..eed4d7e8 100755
--- a/.gitignore
+++ b/.gitignore
@@ -1,2 +1,3 @@
 build/
 out/
+.mill-version
diff --git a/Makefile b/Makefile
index 54309cea..6d76a223 100755
--- a/Makefile
+++ b/Makefile
@@ -3,7 +3,7 @@ V_FILE_FINAL = build/ysyxSoCFull.v
 SCALA_FILES = $(shell find src/ -name "*.scala")
 
 $(V_FILE_FINAL): $(SCALA_FILES)
-	CHISEL_FIRTOOL_PATH=./ mill -i ysyxsoc.runMain ysyx.Elaborate --target-dir $(@D)
+	mill -i ysyxsoc.runMain ysyx.Elaborate --target-dir $(@D)
 	mv $(V_FILE_GEN) $@
 	sed -i -e 's/_\(aw\|ar\|w\|r\|b\)_\(\|bits_\)/_\1/g' $@
 	sed -i '/firrtl_black_box_resource_files.f/, $$d' $@
-- 
2.34.1

