From 10cd21a97120e8385ccddd9101992d4f01cc778d Mon Sep 17 00:00:00 2001
From: PengchengYang-xdu <24241214835@stu.xidian.edu.cn>
Date: Wed, 8 Oct 2025 17:09:44 +0800
Subject: [PATCH 72/72] ci commit 1

---
 .mill-version | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100755 .mill-version

diff --git a/.mill-version b/.mill-version
deleted file mode 100755
index 62d5dbdf..00000000
--- a/.mill-version
+++ /dev/null
@@ -1 +0,0 @@
-0.11.5
-- 
2.34.1

